bot/exts/utils/source.py

```diff
@@ -0,0 +1,133 @@
+import inspect
+from pathlib import Path
+from typing import Optional, Tuple, Union
+
+from discord import Embed, utils
+from discord.ext import commands
+
+from bot.bot import ThreadBot
+from bot.constants import URLs
+
+SourceType = Union[commands.HelpCommand, commands.Command, commands.Cog, str, commands.ExtensionNotLoaded]
+
+
+class SourceConverter(commands.Converter):
+    """Convert an argument into a help command, tag, command, or cog."""
+
+    @staticmethod
+    async def convert(ctx: commands.Context, argument: str) -> SourceType:
+        """Convert argument into source object."""
+        if argument.lower() == "help":
+            return ctx.bot.help_command
+
+        cog = ctx.bot.get_cog(argument)
+        if cog:
+            return cog
+
+        cmd = ctx.bot.get_command(argument)
+        if cmd:
+            return cmd
+
+        tags_cog = ctx.bot.get_cog("Tags")
+        show_tag = True
+
+        if not tags_cog:
+            show_tag = False
+        elif argument.lower() in tags_cog._cache:
```

Comment:
```suggestion
        show_tag = True if tags_cog else False

        if argument.lower() in tags_cog._cache:
```
Which is just..
```suggestion
        show_tag = bool(tags_cog)

        if argument.lower() in tags_cog._cache:
```
Which simply can be removed and just use `if tags_cog`.

Reply:
46bb863